hidapi_fix_prefix.patch

```diff
@@ -0,0 +1,20 @@
+--- a/setup.py	2020-02-07 23:32:40.000000000 +0100
```

Comment:
The patch doesn't seem to be used anywhere

Reply:
Oh, good catch, it was a leftover. Removed.